src/shared/utilities/cliUtils.ts

```diff
@@ -41,13 +43,13 @@ interface Cli {
     name: string
 }
 
-type AwsClis = telemetry.ToolId
+type AwsClis = 'session-manager-plugin'
```

Comment:
Nit: maybe tie this to the telemetry ID
```suggestion
type AwsClis = Pick<telemetry.ToolId, 'session-manager-plugin'>
```